codeReview.js

```diff
@@ -0,0 +1,78 @@
+import { Router, Request, Response } from 'api-framework'; // express-like package
+ 
+   import { getDBConnection } from '../config/db';
+   import { User } from '../types/user';
+   import { sendEmail } from '../util/send_email';
+   import { sendText } from '../util/send_text';
+ 
+   const router = Router();
+   const db = getDBConnection();
+ 
+   const fetchUsersByEmail = async (emails: string[]) => {
+     const users: User[] = await db<User>('users')
+       .select('id', 'email', 'first_name', 'enabled_features')
+       .whereIn('email', emails);
+ 
+     if (!users) {
+       throw new Error('No users found');
+     }
+ 
+     return users;
+   };
+ 
+   router.patch('/features', async (req: Request, res: Response) => {
+     const { emails, feature } = req.body; // these should be a list of emails and a feature, respectively
```

Comment:
add input validation, validate the emails are structured correctly b4 hitting the db, and make sure the feature exists